GLib-2.0.gir

```diff
@@ -7145,32 +7121,6 @@ and %FALSE otherwise.</doc>
         </parameter>
       </parameters>
     </callback>
-    <callback name="EqualFuncFull" c:type="GEqualFuncFull" version="2.74">
```

Comment:
Eh what happened here? @bilelmoussaoui @alatiera any guesses why so much is removed?

Reply:
As @alatiera found out, the reason is: https://gitlab.gnome.org/GNOME/gnome-build-meta/-/commit/ee4352dc93a3b1f50215af6216971308cefe6098

We'll have to wait until the GNOME SDK tracks glib main again before updating anything.

Reply:
We need this on the sdk side to get newer meson https://gitlab.gnome.org/GNOME/gnome-build-meta/-/merge_requests/1589

Reply:
This is fixed now \o/

Reply:
Then let's trigger the CI to get this updated. Let's see what happens and how many annotations we have to fix before this can be merged :)